tc-messaging/src/main/java/com/tc/object/EntityID.java

```diff
@@ -84,9 +84,6 @@ public static EntityID readFrom(TCByteBufferInput serialInput) throws IOExceptio
 
   @Override
   public String toString() {
-    return "EntityID{" +
-           "className='" + className + '\'' +
-           ", entityName='" + entityName + '\'' +
-           '}';
+    return className + ":" + entityName;
   }
```

Comment:
remove this change.  If you want this format in code somewhere just produce it from the component parts.